notebooks/README.md

```diff
@@ -24,33 +24,34 @@ We've laid out the steps to get started with either platform below.
 Pre-requisites:
 
 * [Google Colab](https://research.google.com/colaboratory/faq.html)
-* A Gmail account that you're logged into. This will allow you to use Google Colab, which in the free tier will give you access to an NVidia T4 x 15GB GPU.
+* A Google account that you're logged into. This will allow you to use Google Colab, which in the free tier will give you access to an NVidia T4 x 15GB GPU.
 
 ## Running the notebook
 
+<a target="_blank" href="https://colab.research.google.com/github/instructlab/instructlab/blob/main/notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb">
+  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+</a>
+
 [The notebook](./Training_a_LoRA_With_Instruct_Lab.ipynb) in this folder will walk you through:
 
-1. [Open the notebook in Google Colab](https://colab.research.google.com/github/instructlab/instructlab/blob/main/notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb)
-1. Uploading the output of `ilab generate` (a synthetic dataset created based on your hand written prompts/responses).
-1. Checking the base model before training
-1. Setting up and training a LoRA. A LoRA uses Parameter Efficient Fine-tuning (PEFT) methods to fine-tune a model on a small subset of the overall parameters, allowing you to conduct fine-tuning in a fraction of the time, on a fraction of the hardware required. The resultant model should be updated and better handle your queries than the base model.
-1. Inspecting the output model to make sure the LoRA training had the desired effect (i.e. the output has improved).
+1. Uploading the output of `ilab generate` (a synthetic dataset created based on your hand-written prompts/responses).
+2. Checking the base model before training
+3. Setting up and training a LoRA. A LoRA uses Parameter-Efficient Fine-Tuning (PEFT) methods to fine-tune a model on a small subset of the overall parameters, allowing you to conduct fine-tuning in a fraction of the time, on a fraction of the hardware required. The resultant model should be updated and better handle your queries than the base model.
+4. Inspecting the output model to make sure the LoRA training had the desired effect (i.e. the output has improved).
```

Comment:
```suggestion
1. Uploading the output of `ilab generate` (a synthetic dataset created based on your hand-written prompts/responses).
1. Checking the base model before training
1. Setting up and training a LoRA. A LoRA uses Parameter-Efficient Fine-Tuning (PEFT) methods to fine-tune a model on a small subset of the overall parameters, allowing you to conduct fine-tuning in a fraction of the time, on a fraction of the hardware required. The resultant model should be updated and better handle your queries than the base model.
1. Inspecting the output model to make sure the LoRA training had the desired effect (i.e. the output has improved).
```

Reply:
These should be left as they were - this does automatic numbering in Markdown

Reply:
Ok I will change this. I guess the Jupyter extension of VS Code changed this automatically.

Reply:
Done

Reply:
Thanks!